contracts/interfaces/IAllocationStrategy.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity 0.8.19;
 
+import "@openzeppelin-upgradeable/proxy/utils/Initializable.sol";
```

Comment:
Do we need this ?

Reply:
No, i think we just forgot to remove it.